gateway/cli/src/main.rs

```diff
@@ -101,93 +105,88 @@ async fn main() {
             println!("version: {}", env!("GIT_HASH"));
         }
         Commands::Info => {
-            call(
-                source_password(cli.rpcpassword),
-                cli.url,
-                String::from("/info"),
-                (),
-            )
-            .await;
+            let response = client
+                .get_info(source_password(cli.rpcpassword))
```

Comment:
Eventually we might be able to have authenticated/unauthenticated clients (same struct, just optional password saved in there), for now this is good.